### PR TITLE
Bookstore: initialize project with components

### PR DESCRIPTION
**Pull request by HenaryKhuraijam**
### Bookstore: initialize the project with components
- Installed React Router V6
- Added 2 routes in my application
- Created components inside of the /components directory

### CODE REVIEW

File: src/components/Book.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  <>
+    <div className="row container mx-auto mb-3 px-3 py-3 bg-white d-flex align-items-between justify-content-between">
+      <div className="col-5 m-0 p-0">
+        <div className="mx-0 mb-2 p-0">
+          <h4 className="m-0 p-0 text-2">{title}</h4>
+          <p className="m-0 p-0 text-3">{author}</p>
+        </div>
+        <div className="d-flex align-items-center justify-content-start m-0 p-0">
+          <p className="border-end pe-3 py-0 m-0 text-3">Comments</p>
+          <p className="border-end px-3 py-0 m-0 text-3">Remove</p>
```
Comment (zeff96) [inline src/components/Book.js]: - [ ] Hey there! To make it easier to remove books in the future, let's switch from using a `<p>` tag to a `<button>` element. 📚✨

**Review by zeff96 — CHANGES_REQUESTED** [review]
### Status: CHANGES REQUIRED ♻️ ♻️ ♻️ 
Hi @HenaryKhuraijam , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

close

## Highlights
:heavy_check_mark: Linters passing
:heavy_check_mark: Correct use of GitFlow

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Book.js (hunk 1 above)
Comment (HenaryKhuraijam) [reply src/components/Book.js]: Done

**Review by frank1738 — APPROVED** [review]
Hi @HenaryKhuraijam , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

- [ ] Good work updating your book component

## Optional suggestions

N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**